src/wasm/extract.hpp

```diff
@@ -20,6 +20,8 @@
 
 #include "wasm/fdzipstream/fdzipstream.h"
 
+#define wasm_debug(X) do { if (extractor::get().get_settings().debug_messages_enabled) log_info << X; } while(0)
```

Comment:
Since you use inside log_info I would recommend to change this macro name to: wasm_info